util/src/main/java/org/folio/metastorage/util/XmlJsonUtil.java

```diff
@@ -25,170 +25,15 @@
 import javax.xml.transform.stream.StreamSource;
 import org.apache.logging.log4j.LogManager;
 import org.apache.logging.log4j.Logger;
-import org.w3c.dom.Document;
-import org.w3c.dom.Element;
-import org.w3c.dom.Node;
-import org.w3c.dom.NodeList;
-import org.xml.sax.InputSource;
-import org.xml.sax.SAXException;
+
 
 public final class XmlJsonUtil {
```

Comment:
It would be clean to split up this class into three util classes:
* general XML/JSON utilities
* Marc-in-JSON utilities
* FOLIO Inventory utilities (probably no longer needed)

Reply:
Done. The Inventory stuff is still there as the client can still do XSLT.. But I won't miss the "inventory XML". It was actually not easy to make that conversion. oh well.